# Aufgabe 24

## a)

Designmatrix aufstellen: 
\begin{equation}
A= \begin{pmatrix}
  \cos{(\frac{0}{6}\pi)} & \sin{(\frac{0}{6}\pi)} \\
  \cos{(\frac{1}{6}\pi)} & \sin{(\frac{1}{6}\pi)} \\
  \cos{(\frac{2}{6}\pi)} & \sin{(\frac{2}{6}\pi)} \\
  \cos{(\frac{3}{6}\pi)} & \sin{(\frac{3}{6}\pi)} \\
  \cos{(\frac{4}{6}\pi)} & \sin{(\frac{4}{6}\pi)} \\
  \cos{(\frac{5}{6}\pi)} & \sin{(\frac{5}{6}\pi)} \\
  \cos{(\frac{6}{6}\pi)} & \sin{(\frac{6}{6}\pi)} \\
  \cos{(\frac{7}{6}\pi)} & \sin{(\frac{7}{6}\pi)} \\
  \cos{(\frac{8}{6}\pi)} & \sin{(\frac{8}{6}\pi)} \\
  \cos{(\frac{9}{6}\pi)} & \sin{(\frac{9}{6}\pi)} \\
  \cos{(\frac{10}{6}\pi)} & \sin{(\frac{10}{6}\pi)} \\
  \cos{(\frac{11}{6}\pi)} & \sin{(\frac{11}{6}\pi)} \\
\end{pmatrix}
\end{equation}


## b)

In [2]:
import numpy as np


\begin{equation}
    \hat{a}=\left(A^{T}A \right)^{-1} A^{T} y
\end{equation}

In [3]:
A=np.matrix([[np.cos(0), np.sin(0)],
            [np.cos(1/6*np.pi), np.sin(1/6*np.pi)], 
            [np.cos(2/6*np.pi), np.sin(2/6*np.pi)], 
            [np.cos(3/6*np.pi), np.sin(3/6*np.pi)], 
            [np.cos(4/6*np.pi), np.sin(5/6*np.pi)],
            [np.cos(5/6*np.pi), np.sin(5/6*np.pi)], 
            [np.cos(6/6*np.pi), np.sin(6/6*np.pi)], 
            [np.cos(7/6*np.pi), np.sin(7/6*np.pi)], 
            [np.cos(8/6*np.pi), np.sin(8/6*np.pi)], 
            [np.cos(9/6*np.pi), np.sin(9/6*np.pi)], 
            [np.cos(10/6*np.pi), np.sin(10/6*np.pi)], 
            [np.cos(11/6*np.pi), np.sin(11/6*np.pi)]])

y=np.matrix([-0.032, 0.010, 0.057, 0.068, 0.076, 0.080, 0.031, 0.005, -0.041, -0.041 -0.090, -0.088, -0.074])

a=(A.T*A)**(-1)*A.T*y.T
a

matrix([[-0.04019568],
        [ 0.08817037]])

## C)

\begin{equation}
V[\hat{a}]= \sigma^2 \left(A^T A\right)
\end{equation}

In [4]:
sigma = 0.011
V = sigma**2 * (A.T * A)
V

matrix([[7.26000000e-04, 2.21445369e-05],
        [2.21445369e-05, 6.65500000e-04]])

Also ergibt sich:
\begin{align}
    \sigma_{\alpha_1} &= 0,0294 \\
    \sigma_{\alpha_2} &= 0,0258
\end{align}

In [5]:
rho=V[0, 1]/(np.sqrt(V[0,0]*V[1, 1]))
rho

0.03185842215551651

womit
\begin{equation}
    \rho= \frac{\sigma_{1, 2}}{\sigma_1 \sigma_2} = \frac{2.2145 \cdot 10^{-5}}{\sqrt{7,2600 \cdot 10^{-4} \cdot 6,6550 \cdot 10^{-4}}}= 0,0319
\end{equation}
folgt.


## d)

Es gilt:
\begin{equation}
    \cos{(\Psi+\delta)}=\cos{(\Psi)} \cos{(\delta)} - \sin{(\Psi)} \sin{(\delta)}
\end{equation}

Daraus folgt:
\begin{align*}
    a_1 &= A_0 \cdot \cos{\delta} \\
    a_2 &= A_0 \cdot \sin{\delta} \\
      \rightarrow  {a_1}^2+{a_2}^2={A_0}^2 & \rightarrow A_0=\sqrt{{a_1}^2+{a_2}^2}
\end{align*}